recipe/meta.yaml

```diff
@@ -28,6 +27,19 @@ test:
     - rest_framework.templatetags
     - rest_framework.utils
 
+outputs:
+  - name: djangorestframework
+    build:
+      script: python -m pip install --no-deps --ignore-installed .
+    requirements:
+      - python
+      - django
+
+  - name: django-rest-framework
```

Comment:
https://pypi.org/project/django-rest-framework/ looks like an old version and not really an alias. Are you sure this is needed? Looks like an abandoned name.

django-rest-framework 0.1.0
djangorestframework 3.8.2

Reply:
It is defintinely needed for me.  I have mistyped this name in this way >10x at this point :smile:

Reply:
If I understand this correctly the alias is not an official feature on PyPI.

The package was renamed to `djangorestframework` and the old name `django-rest-framework` was abandoned! There won't be 1-1 correspondence with PyPI b/c `pip install django-rest-framework` will install a different version of `django-rest-framework` than the one you are proposing here. That can bring even more confusion to the table.

Do you really want support an old name that was abandoned by upstream?

I'll leave this one to @kwilcox to decide.

Reply:
`django-rest-framework` is an ugly hack and you really shouldn't be using it. It's an "empty" package that requires any version of `djangorestframework`.

```
setup(
    name='django-rest-framework',
    version='0.1.0',
    url='http://www.django-rest-framework.org',
    license='BSD',
    description='alias.',
    author='Tom Christie',
    author_email='tom@tomchristie.com',
    packages=['empty'],
    install_requires=['djangorestframework'],
    zip_safe=False,
    classifiers=[
        'Topic :: Internet :: WWW/HTTP',
    ]
)
```

Reply:
right, so this is actually better than the version on pypi because the version numbers will match, but will have the same intended result.

Reply:
> It's an "empty" package that requires any version of `djangorestframework`

That explains the version difference :smile:

Reply:
I don't have any strong feelings on this either way but it does seem like a good chance to prevent bad practice by **not** supplying a `django-rest-framework` package. Is it a difficult process to update your dependencies to the correct name?

Reply:
Why is it a bad practice to support package aliasing? And this name isn't abandoned, which is part of the problem. django-rest-framework.org is their url, they use it all throughout their docs, and they made an alias on pypi. Also, it is highly unlikely that any other package would ever choose this name. Additionally because anaconda.org search is not great, this makes it seem like DRF isn't provided by conda-forge when you search with dashes.

So I am saying that, as a user, the situation is very confusing and annoying. I figure if I am having trouble with it there are others who likely will too.

Reply:
In this case we would be diverging from PyPI by providing a package with the same name but with different versions. I'd argue that is more confusing to users than an (arguably) incorrect package name not existing in `conda-forge`. I'll merge if you can find an existing package in `conda-forge` that does something similar but I'm not willing to set a precedent here.

Reply:
There are many cases where the PyPI name does not match the conda-forge name. `ssl_match_hostname` is called `backports.ssl_match_hostname` on PyPI. https://github.com/conda-forge/ssl_match_hostname-feedstock, for example

Reply:
https://github.com/conda-forge/ssl_match_hostname-feedstock/issues/1